app/lib/health_check/search_check_report.rb

```diff
@@ -0,0 +1,28 @@
+require 'csv'
+
+module HealthCheck
+  class SearchCheckReport
```

Comment:
if we are going to move this can we change this class to write to a db as well as a csv

Reply:
This is the next step. I don't think it's small enough to do in this PR though. I may pick it up if there's spare time over christmas.